products/jbrowse-cli/src/commands/admin-server.ts

```diff
@@ -1,7 +1,7 @@
 import { flags } from '@oclif/command'
 import { promises as fsPromises } from 'fs'
-import * as path from 'path'
-import * as express from 'express'
+import path from 'path'
```

Comment:
This change makes `path` be `undefined` for me when running with `./bin/run`

Reply:
See above